README.md

```diff
@@ -1,2 +1,52 @@
 # pwa-prototype-reporting
-A playground for a PWA for reporting only purposes. 
+A playground for a PWA for reporting only purposes. This project started as an experimental prototype to migrate the functionality of Ushahidi survey submissions into a small standalone project that can be deployed anywhere and in the future could be linked to any ushahidi.io deployment chosen by the user.
+
+# Offline support
+- Supports loading the form while offline if you have used the app in the past
+- Saving form responses while offline and syncing the response with the backend when the user is online
+
+# Dynamic attribute loading
+- Loads the deployment attributes for any ushahidi v3 form
+
+# Getting set up for development
+
+## Libraries/Packages/Dependencies
+
+First you'll need nodejs installed,
+npm takes care of the rest of our dependencies.
+
+* nodejs v6
```

Comment:
why are we stating that node v6 is required? This works just fine with the current node LTS (10.x)  as it uses a modern stack

Reply:
Made the changes.